packages/flutter/test/widgets/page_view_test.dart

```diff
@@ -565,88 +565,6 @@ void main() {
     expect(tester.getTopLeft(find.text('Hawaii')), const Offset(-100.0, 0.0));
   });
 
-  testWidgets(
```

Comment:
Is this one a test case that's worth keeping since this is expected to work in either implementation?